troubleshooting.md

```diff
@@ -424,9 +431,9 @@ Choose one of the following:
     * Install the fuse-overlayfs package for your Linux Distribution.
     * Add `mount_program = "/usr/bin/fuse-overlayfs"` under `[storage.options]` in your `~/.config/containers/storage.conf` file.
 
-### 17) rhel7-init based images don't work with cgroups v2
+### 17) RHEL 7 and CentOS 7 based `init` images don't work with cgroups v2
 
-The systemd version shipped in rhel7-init doesn't have support for cgroups v2.  You'll need at least systemd 230.
+The systemd version shipped in RHEL 7 and CentOS 7 doesn't have support for cgroups v2.  Cgroup V2 support requires at least systemd 230.
```

Comment:
cgroup is supposed to be lower case everywhere.  How about the second sentence starts with "Further, cgroup v2".   Regardless, "V2" to "v2" for consistency.